### DISCUSSION TITLE
Model loading question

### DISCUSSION THREAD

**Discussion by ghost** [LostRuins/koboldcpp#166] [Q&A] [answered]
Hey there, gonna tag someone directly in this question here as they would 100% know for sure. @LostRuins. 

All the documentation I have read talks about pointing to a model.bin file. However I am seeing more and more models popup with the model.safetensors file format. Can/would Koboldcpp be able to work and support pointing and reading from the safetensors file format rather than a bin file format?

Is this something I should request upstream in llama?

**Comment by LostRuins** [comment, marked as answer]
Can't load safetensors directly. You'd have to convert it which  should be possible
